src/github.com/matrix-org/dendron/proxy/paths.go

```diff
@@ -0,0 +1,146 @@
+package proxy
+
+// Note that this file was generated by running `python dendron/scripts/generate_proxy_paths.py`.
+// It should not be edited by hand.
+
+import "regexp"
+
+type regexpAndString struct {
+	re *regexp.Regexp
+	s  string
+}
+
+var (
+	constantPaths = make(map[string]bool)
+	regexpPaths   []regexpAndString
+)
+
+func endpointFor(path string) string {
+	if _, ok := constantPaths[path]; ok {
```

Comment:
Doesn't really make a difference. Might as well switch to ok := ... because it's a little more concise, and also then if someone adds a false value in for some reason it will work...

Done